homeassistant/components/sensor/brottsplatskartan.py

```diff
@@ -0,0 +1,136 @@
+"""Sensor for Brottsplatskartan."""
+from collections import defaultdict
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    ATTR_ATTRIBUTION,
+    ATTR_LATITUDE,
+    ATTR_LONGITUDE,
+    CONF_LATITUDE,
+    CONF_LONGITUDE,
+    CONF_NAME,
+)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+REQUIREMENTS = ['brottsplatskartan==0.0.1']
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_AREA = 'area'
+DEFAULT_NAME = 'Brottsplatskartan'
+DOMAIN = 'brottsplatskartan'
```

Comment:
The domain of this platform is `'sensor' `.

Reply:
Updated the code.